specification/trustedsigning/TrustedSigning/tspconfig.yaml

```diff
@@ -12,29 +12,29 @@ options:
     emit-lro-options: "none"
     emitter-output-dir: "{project-root}/.."
     omit-unreachable-types: true
-    output-file: "{azure-resource-provider-folder}/Azure.Developer.Signing/{version-status}/{version}/azure.developer.signing.json"
+    output-file: "{azure-resource-provider-folder}/Azure.Developer.TrustedSigning/{version-status}/{version}/azure.developer.trustedsigning.json"
   "@azure-tools/typespec-python":
-    package-dir: "azure-developer-signing"
+    package-dir: "azure-developer-trustedsigning"
     package-name: "{package-dir}"
     flavor: azure
     generate-test: true
     generate-sample: true
     examples-dir: "{project-root}/examples"
   "@azure-tools/typespec-csharp":
     clear-output-folder: true
-    package-dir: "Azure.Developer.Signing"
+    package-dir: "Azure.Developer.TrustedSigning"
     model-namespace: false
     namespace: "{package-dir}"
     flavor: azure
   "@azure-tools/typespec-ts":
-    package-dir: "azure-developer-signing-rest"
+    package-dir: "azure-developer-trustedsigning-rest"
     generateMetadata: true
     generateTest: true
     packageDetails:
-      name: "@azure-rest/developer-signing"
-      description: "Azure developer signing service"
+      name: "@azure-rest/developer-trusted signing"
```

Comment:
@Jaxelr Is the name here `@azure-rest/developer-trustedsigning` or `@azure-rest/developer-trusted-signing`?

Reply:
it should be the first @azure-rest/developer-trustedsigning